Framework/Algorithms/src/InterpolatingRebin.cpp

```diff
@@ -200,27 +201,18 @@ void InterpolatingRebin::outputYandEValues(
 *  @throw invalid_argument if any output x-values are outside the range of input
 *x-values
 **/
-void InterpolatingRebin::cubicInterpolation(const HistogramData::BinEdges &xOld,
-                                            const MantidVec &yOld,
-                                            const MantidVec &eOld,
-                                            const HistogramData::BinEdges &xNew,
-                                            MantidVec &yNew,
-                                            MantidVec &eNew) const {
-  // Make sure y and e vectors are of correct sizes
+void InterpolatingRebin::cubicInterpolation(const Histogram &oldHistogram,
```

Comment:
@SimonHeybrock I tried doing it the way you recommended in the previous comment `Histogram cubicInterpolation(const Histogram &histogram, const BinEdges &xNew)`, but I wan't sure how to create Y and E after that, because they base their size on `outputW->mutableY` and `outputW->mutableE`